packages/synapse-react-client/src/components/SchemaDrivenAnnotationEditor/SchemaDrivenAnnotationEditor.tsx

```diff
@@ -98,6 +100,7 @@ export function SchemaDrivenAnnotationEditor(
     formRef: formRefFromParent,
     onChange = noop,
     hideActions = false,
+    onFormDirtyChange,
```

Comment:
Setting the default value to `noop` lets you get rid of weird nullish checks like `onFormDirtyChange?.(...)`

```suggestion
    onFormDirtyChange = noop,
```

Reply:
Removed